onyo/lib/commands.py

```diff
@@ -437,7 +437,10 @@ def onyo_mv(inventory: Inventory,
             inventory.rename_directory(sources[0], destination)
 
     if inventory.operations_pending():
-        ui.print('The following will be moved:')
+        if subject == "mv":
+            ui.print('The following will be moved:')
+        else:
+            ui.print('The following will be renamed:')
```

Comment:
Minor suggestion, but for such minimal thing I'd prefer a one-liner.
```suggestion
        ui.print("The following will be {}.".format("moved" if subject == "mv" else "renamed"))
```